data-prepper-api/src/main/java/com/amazon/dataprepper/model/metric/Histogram.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.model.metric;
+
+import java.util.List;
+
+/**
+ * A Histogram event
+ *
+ * @since 1.4
+ */
+public interface Histogram extends Metric {
+
+    /**
+     * Gets the sum for the histogram
+     * @return the sum of the values in the population
+     * @since 1.4
+     */
+    Double getSum();
+
+    /**
+     * Gets the bucket count for the histogram
+     * @return the bucket count
+     * @since 1.4
+     */
+    Integer getBucketCount();
+
+    /**
+     * Gets the number of explicit bounds for the histogram
+     * @return the number of bounds
+     * @since 1.4
+     */
+    Integer getExplicitBoundsCount();
+
+    /**
+     * Gets the aggregation temporality for the histogram
+     * @return the aggregation temporality
+     * @since 1.4
+     */
+    String getAggregationTemporality();
+
+    /**
+     * Gets the actual buckets for a histogram
+     * @return the buckets
+     * @since 1.4
+     */
+    List<JacksonHistogram.Bucket> getBuckets();
```

Comment:
This interface shouldn't have any reference to the Jackson implementations. We can follow the pattern that the trace model uses for `TraceGroupFields`. It has an interface for `TraceGroupFields` and then an implementation `DefaultTraceGroupFields`.

Also, Java supports static inner classes within interfaces. If you do not believe there is much value in using an interface here, we could move the `Bucket` class into `Histogram` from `JacksonHistogram`.

Reply:
Thanks @dlvenable ! We missed this one, also the Quantile. We'll address this and the other review comments in the next commits

Reply:
We added a Bucket interface and a default implementation, as suggested